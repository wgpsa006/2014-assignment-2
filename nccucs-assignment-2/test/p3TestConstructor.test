# Test that the constructor returns an empty list of nodes
CreateGraph A
CreateGraph B
ListNodes A
ListNodes B