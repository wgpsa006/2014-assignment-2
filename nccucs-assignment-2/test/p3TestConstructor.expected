# Test that the constructor returns an empty list of nodes
created graph A
created graph B
A contains:
B contains: